Lib/fontTools/otlLib/builder.py

```diff
@@ -491,6 +493,14 @@ def _makeDeviceTuple(device):
     return (device.DeltaFormat, device.StartSize, device.EndSize,
             tuple(device.DeltaValue))
 
+def _getSinglePosValueSize(valueKey):
+    count = 0
+    for k in valueKey[1:]:
+        if hasattr(k[1], '__len__') and len(k[1]):
+            count += len(k[1][3]) + 3
+        else:
+            count += 1
+    return count
```

Comment:
I'd suggest to give more descriptive names to the variables or add comments, because it's not immediately clear what valueKey is here, and what it contains

Reply:
It's hard to come up with another name given `valueKey` comes from `_getSinglePosValueKey` which kind of introduces a ValueKey as a concept in the code. Hence using that name here. I've added a comment to the function to help clarify things a little.